instaclustr/resource_cluster.go

```diff
@@ -687,16 +737,33 @@ func resourceClusterRead(d *schema.ResourceData, meta interface{}) error {
 	if len(cluster.DataCentres[0].ResizeTargetNodeSize) > 0 {
```

Comment:
I don't think this will work with multi-dc. Probably anything using DataCentres[0] is probably not going to be directly compatible for multi-dc as you are just sampling 1 of the many dcs in the cluster